templates/CRM/Member/Form/MembershipRenewal.tpl

```diff
@@ -303,5 +402,37 @@
         cj('#record-different-contact').hide();
       }
     }
+
+    function buildAmount( ) {
```

Comment:
Same comment here.  The two have similarities, but are still significantly different.